pkgs/servers/elasticmq-server-bin/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, jdk, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "elasticmq-server-${version}";
+  version = "0.14.6";
+
+  src = fetchurl {
+    url = "https://s3-eu-west-1.amazonaws.com/softwaremill-public/${name}.jar";
+    sha256 = "1cp2pmkc6gx7gr6109jlcphlky5rr6s1wj528r6hyhzdc01sjhhz";
+  };
+
+  # don't do anything?
+  unpackPhase = "${jdk}/bin/jar xf $src favicon.png";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/elasticmq-server
+
+    cp $src $out/share/elasticmq-server/elasticmq-server.jar
+
+    # TODO: how to add extraArgs? currently workaround is to use JAVA_TOOL_OPTIONS environment to specify properties
+    makeWrapper ${jre}/bin/java $out/bin/elasticmq-server \
+      --add-flags "-jar $out/share/elasticmq-server/elasticmq-server.jar"
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/softwaremill/elasticmq;
```

Comment:
```suggestion
    homepage = "https://github.com/softwaremill/elasticmq";
```

Reply:
hmm others told me before to remove quotes from `homepage` :/

but well

Reply:
oh yeah, there is a little movement around this, sorry: https://github.com/NixOS/rfcs/pull/45